src/textual/renderables/sparkline.py

```diff
@@ -15,7 +15,7 @@
 SummaryFunction = Callable[[Sequence[T]], float]
 
 
-class Sparkline(Generic[T]):
+class SparklineRenderable(Generic[T]):
```

Comment:
Not keen on the redundant `Renderable` in the name. Suggest when you import it you can do `from renderables.sparkline import Sparkline as SparklineRenderable`